csharp_dotnetcore/3.Welcome-Users/Program.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+
+namespace WelcomeUser
+{
+    public class Program
```

Comment:
This is what I used for this file:

```cs
    /// <summary>
    /// This is an ASP.NET Core app that creates a webserver.
    /// </summary>
    public class Program
    {
        /// <summary>
        /// This is the entry point for your application.
        /// It is run first once your application is started.
        /// This method creates a web server.
        /// </summary>
        /// <param name="args">Arguments for this method.</param>
        public static void Main(string[] args)
        {
            BuildWebHost(args).Run();
        }

        /// <summary>
        /// Initializes a new instance of the <see cref="T:Microsoft.AspNetCore.Hosting.WebHostBuilder" />
        /// class with pre-configured defaults.
        /// The UseStartup method on <see cref="IWebHost"/> specifies the Startup class for your app.
        /// </summary>
        /// <param name="args">Arguments for this method.</param>
        /// <returns>A web server.</returns>
        public static IWebHost BuildWebHost(string[] args) =>
            WebHost.CreateDefaultBuilder(args)
                .UseStartup<Startup>()
                .Build();
    }
```

Reply:
I snapped to the work @daveta  did with his #301. Feels as if we need to align and make a final pass on comments across the board?

Reply:
@daveta & I went through this and aligned to the samples the ASP.Net Core team did. They didn't include comments in this file in the majority of their samples, so we followed suit.